02_plotting-mutation-counts.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "Plotting Gene Mutation Counts"
+author: Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+---
+
+## Introduction
+
+We at the Data Lab really like using notebook documents like [Rmarkdown](https://rmarkdown.rstudio.com) (this document) and [Jupyter](https://jupyter.org) for development and data exploration.
+Notebooks allow mixing of text, code, and results in a single document that is easy to work with interactively.
+This allows much more freedom to include a lot of explanatory text, much like code comments, but with simpler syntax and the ability to include formatting and links more easily through the use of [markdown-formatted text](https://rmarkdown.rstudio.com/lesson-8.html) (or using the new visual editor in RStudio).
```

Comment:
I'm a little/extremely pedantic about always using a noun after "this," but I'm not offended if you don't want to. 

```suggestion
Using notebooks allows much more freedom to include a lot of explanatory text, much like code comments, but with simpler syntax and the ability to include formatting and links more easily through the use of [markdown-formatted text](https://rmarkdown.rstudio.com/lesson-8.html) (or using the [new visual editor in RStudio](https://www.rstudio.com/blog/exploring-rstudio-visual-markdown-editor/)).
```